code/game/objects/structures/flora.dm

```diff
@@ -222,6 +222,44 @@
 	icon_state = "plant-01"
 
 	plane = OBJ_PLANE
+	var/obj/item/stored_item
+	
+/obj/structure/flora/pottedplant/examine(mob/user)
+	..()
+	if(in_range(user, src) && stored_item)
+		to_chat(user, "<i>You can see something in there...</i>")
+
+/obj/structure/flora/pottedplant/attackby(obj/item/I, mob/user)
+	if(stored_item)
+		to_chat(user, "<span class='notice'>[I] won't fit in. There already appears to be something in here...</span>")
+		return
+
+	if(I.w_class > ITEMSIZE_SMALL)

```

Comment:
> ITEMSIZE_SMALL allows small items into the pots

Reply:
Would you prefer if it was ITEMSIZE_TINY for balancing?

Reply:
You noted they could only hold tiny items in the PR desc, was merely pointing out an inconsistency. I don't have any qualms either way.

Reply:
Yeah I amended it, let's see how it goes. I hope tape recorders can fit.